<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tea_EvaluateTeacherWindow</class>
 <widget class="QWidget" name="tea_EvaluateTeacherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>评教结果窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="evaluatetableView"/>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>项目</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>评教总分</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>教学态度</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>教学内容</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>教学方法</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>教学效果</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <item>
       <property name="text">
        <string>平均分</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最高分</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>最低分</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>占比</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QTabWidget" name="teacherTabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabBarView">
       <attribute name="title">
        <string>柱状图</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" colspan="2">
         <widget class="QChartView" name="teacherBarView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPieView">
       <attribute name="title">
        <string>饼状图</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0" colspan="2">
         <widget class="QChartView" name="teacherPieView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>分析</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/my photos/way.png</normaloff>:/Image/my photos/way.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1120</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
